packages/chap-lib/src/components/maps/Legend.tsx

```diff
@@ -0,0 +1,26 @@
+
+import React from 'react';
+import styles from './Legend.module.css';
+
+const Legend = ({ bins, colors } : any) => {
+    console.log('Legend:')
```

Comment:
Cleaned up unncessary logging elsewhere, but this one might be nice for debugging, for now? Or should we stick to no logs as much as possible? @turban @hermantretteteig

Reply:
Fine with me, maybe reduce it to one line:
console.log('Legend:', bins, colors)